libyul/optimiser/SSAValueTracker.cpp

```diff
@@ -33,6 +33,16 @@ void SSAValueTracker::operator()(Assignment const& _assignment)
 		m_values.erase(var.name);
 }
 
+set<YulString> SSAValueTracker::ssaVariables(Block const& _ast)
+{
+	SSAValueTracker t;
+	t(_ast);
+	set<YulString> ssaVars;
+	for (auto const& value: t.values())
```

Comment:
```suggestion
	for (auto const& [variableName, value]: t.values())
```
But we might get "unused" warnings/errors for ``value`` and unfortunately there's no way to just skip one at the moment, so maybe not worth it.

Reply:
Yep, I had it like that before. That's really annoying.